sqrl-testing/sqrl-integration-tests/src/test/java/com/datasqrl/AbstractUseCaseTest.java

```diff
@@ -51,6 +51,9 @@ void testUsecase(Path script, Path graphQlFile, Path packageFile) {
     arguments.add("-t"); arguments.add(deployDir.toString());
     arguments.add("--profile");
     arguments.add(getProjectRoot(script).resolve("profiles/default").toString());
+    // TODO: instead of doing this, we should use package.json's profile
+    arguments.add("--profile");
+    arguments.add(getProjectRoot(script).resolve("profiles/postgres_log").toString());
```

Comment:
We previously discussed that this section shouldn't be overly customized. I was considering if we could utilize what's defined in the package.json. However, to make it work, we need to enable the use of relative paths in the package.json's profile definition, which is currently not possible. I also feel that this is beyond the current scope and should be addressed as a follow-up or separate task.

Reply:
Removed